bundles/org.openhab.ui/web/src/components/widgets/generic-widget-component.vue

```diff
@@ -13,10 +13,21 @@
   <div v-else-if="componentType && componentType === 'Label' && visible" :class="config.class" :style="config.style">
     {{ config.text }}
   </div>
+  <fragment v-else-if="componentType && componentType === 'Content'">
+    {{ config.text }}
+  </fragment>
   <pre v-else-if="componentType && componentType === 'Error' && visible" class="text-color-red" style="white-space: pre-wrap">{{ config.error }}</pre>
+  <component v-else-if="visible" :is="componentType" :class="config.class" :style="config.style" @command="onCommand">
```

Comment:
```suggestion
  <component v-else-if="visible" :is="componentType" v-bind="config">
```

> To do this up right, I had to not bind the entire config

Why not?
Having all attributes available is a must for SVG and yes, even the events work - not that I'm particularly happy about that indeed, but well... We could bind an object with all keys beginning with `on` removed but let's leave it for now.

> In the end I moved the key for this from text to content

Good call - it's true that it will add a `content` attribute to the element in addition to putting it before the slots but it's not that much of a problem, `content` as an attribute is only used in `<meta>` - https://www.w3schools.com/tags/att_content.asp

Example:
```yaml
uid: new_expressions
tags: []
props:
  parameters:
    - description: Disable the button
      label: Disabled
      name: disabled
      required: false
      type: BOOLEAN
  parameterGroups: []
timestamp: Oct 11, 2022, 2:05:10 PM
component: button
config:
  class:
   - padding
   - margin
  style:
    width: 200px
  type: button
  content: Button is
  disabled: =props.disabled
  onclick: alert('hmm not sure this is good')
slots:
  default:
    - component: span
      config:
        content: =props.disabled?'Disabled':'Enabled'
        style:
          font-size: 20px
          color: =props.disabled?'red':'green'
```

Reply:
Ok, sounds good.  If you're not concerned about the content attribute, then this is the better way to go.

> I realize I pointed you to the wrong part of script-editor.vue above, but you figured it out :)

The trickiest part was actually realizing I also had to modify `filterPartialCompletions` in hint-utils.js.  I got everything else working quickly and then spent an hour trying to figure out why my hint list kept coming back empty...